src/api/db/migrate/20240111231639_add_theme_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddThemeToUser < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :color_theme, :string, limit: 30, default: "light"
```

Comment:
Make it an `integer` and then enum in the model in the next PR. I would also vote the default to be `system` and not `light` (though for integer just set it to 0 and think about what values in the enum will mean)
```suggestion
    add_column :users, :color_theme, :integer, default: 0, null: false
```

Reply:
Done here, please review again